src/pages/_app.tsx

```diff
@@ -29,6 +31,13 @@ const App = ({ Component, pageProps }: AppProps): ReactNode => {
     useLandingSource()
     useLogAllLinkClicks()
 
+    let telemetryRecorder = noOpTelemetryRecorder
+    if (typeof window !== 'undefined') {
```

Comment:
This is seemingly required to prevent this code from running on the server, a well known workaround for next js: https://stackoverflow.com/questions/55151041/window-is-not-defined-in-next-js-react-app